js/apps/admin-ui/src/realm-settings/GeneralTab.tsx

```diff
@@ -251,79 +247,40 @@ export const RealmSettingsGeneralTab = ({
             )}
           />
         </FormGroup>
-        {isFeatureEnabled(Feature.DeclarativeUserProfile) && (
-          <FormGroup
-            hasNoPaddingTop
-            label={t("userProfileEnabled")}
```

Comment:
Since this label is no longer used it can be removed from the properties file containing the translations.